lib/server.js

```diff
@@ -1,9 +1,48 @@
 'use strict';
 
-var sync = require('fh-wfm-sync/lib/server')
-  , config = require('./config');
+var sync = require('fh-wfm-sync/lib/server'),
+  config = require('./config'),
+  shortid = require('shortid');
 
 
 module.exports = function(mediator, app, mbaasApi) {
   sync.init(mediator, mbaasApi, config.datasetId, config.syncOptions);
+
+  mediator.subscribe(config.topicPrefix + config.datasetId + ':create', function(workflowToCreate, mediatorTopicIdentifier) {
+    // Add an id field required by the new simple store module to the workflow object that will be created
+    workflowToCreate.id = shortid.generate();
+
+    mediator.request(config.topicPrefix + 'data:' + config.datasetId + ':create', workflowToCreate, {uid: workflowToCreate.id})
```

Comment:
It would be good to have a small function to generate topics to prevent errors in concatentation.